addons/hw_drivers/tools/helpers.py

```diff
@@ -286,25 +284,15 @@ def get_mac_address():
 def get_path_nginx():
     return str(list(Path().absolute().parent.glob('*nginx*'))[0])
 
-def get_ssid():
-    ap = subprocess.call(['systemctl', 'is-active', '--quiet', 'hostapd']) # if service is active return 0 else inactive
-    if not ap:
-        return subprocess.check_output(['grep', '-oP', '(?<=ssid=).*', '/etc/hostapd/hostapd.conf']).decode('utf-8').rstrip()
-    process_iwconfig = subprocess.Popen(['iwconfig'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    process_grep = subprocess.Popen(['grep', 'ESSID:"'], stdin=process_iwconfig.stdout, stdout=subprocess.PIPE)
-    return subprocess.check_output(['sed', 's/.*"\\(.*\\)"/\\1/'], stdin=process_grep.stdout).decode('utf-8').rstrip()
-
 
 @cache
 def get_odoo_server_url():
     """Get the URL of the linked Odoo database.
-    If the IoT Box is in access point mode, it will return ``None`` to avoid
-    connecting to the server.
 
     :return: The URL of the linked Odoo database.
     :rtype: str or None
     """
-    return None if access_point() else get_conf('remote_server')
+    return get_conf('remote_server')
```

Comment:
in case of an access point configuration this change will potentially provoke a lot of request errors